tests/functional/u/unbalanced_tuple_unpacking.py

```diff
@@ -121,10 +137,21 @@ def my_sum(self):
 
     def sum_unpack_3_into_4(self):
         """Attempt to unpack 3 variables into 4"""
-        first, second, third, fourth = self # [unbalanced-tuple-unpacking]
+        first, second, third, fourth = self  # [unbalanced-tuple-unpacking]
         return first + second + third + fourth
 
     def sum_unpack_3_into_2(self):
         """Attempt to unpack 3 variables into 2"""
-        first, second = self # [unbalanced-tuple-unpacking]
+        first, second = self  # [unbalanced-tuple-unpacking]
         return first + second
+
+
+def my_function(mystring):
+    """The number of items on the right-hand-side of the assignment to this function is not known"""
+    mylist = []
+    for item in mystring:
+        mylist.append(item)
+    return mylist
+
+
+a, b = my_function("12")  # [unbalanced-tuple-unpacking]
```

Comment:
```suggestion
a, b = my_function("12")  # [unbalanced-tuple-unpacking]
c = my_function("12")
d, *_ = my_function("12") 
```
Checking that there are no false positives, I suppose those should not raise either, right ?

Reply:
Looks sensible; will get back to this asap.